bundles/org.openhab.binding.automower/src/main/java/org/openhab/binding/automower/internal/AutomowerBindingConstants.java

```diff
@@ -154,18 +149,17 @@ public class AutomowerBindingConstants {
     // Messages Channel ids
     public static final String GROUP_MESSAGE = "message#";
 
-    public static final ArrayList<String> CHANNEL_MESSAGE = new ArrayList<>(
-            List.of("msg-timestamp", "msg-code", "msg-text", "msg-severity", "msg-gps-position"));
-
-    public static final ArrayList<ChannelTypeUID> CHANNEL_TYPE_MESSAGE = new ArrayList<>(List.of(
-            new ChannelTypeUID(BINDING_ID, "messageTimeType"), new ChannelTypeUID(BINDING_ID, "messageCodeType"),
-            new ChannelTypeUID(BINDING_ID, "messageType"), new ChannelTypeUID(BINDING_ID, "messageSeverityType"),
-            new ChannelTypeUID(BINDING_ID, "messagePositionType")));
+    public static final String CHANNEL_MESSAGE_TIMESTAMP = GROUP_MESSAGE + "msg-timestamp";
+    public static final String CHANNEL_MESSAGE_CODE = GROUP_MESSAGE + "msg-code";
+    public static final String CHANNEL_MESSAGE_TEXT = GROUP_MESSAGE + "msg-text";
+    public static final String CHANNEL_MESSAGE_SEVERITY = GROUP_MESSAGE + "msg-severity";
+    public static final String CHANNEL_MESSAGE_GPS_POSITION = GROUP_MESSAGE + "msg-gps-position";
```

Comment:
See other prefix comment in readme.md
```suggestion
    public static final String CHANNEL_MESSAGE_TIMESTAMP = GROUP_MESSAGE + "timestamp";
    public static final String CHANNEL_MESSAGE_CODE = GROUP_MESSAGE + "code";
    public static final String CHANNEL_MESSAGE_TEXT = GROUP_MESSAGE + "text";
    public static final String CHANNEL_MESSAGE_SEVERITY = GROUP_MESSAGE + "severity";
    public static final String CHANNEL_MESSAGE_GPS_POSITION = GROUP_MESSAGE + "gps-position";
```